cmd/utils/flags.go

```diff
@@ -1024,6 +1024,40 @@ Please note that --` + MetricsHTTPFlag.Name + ` must be set to start the server.
 		Value:    metrics.DefaultConfig.InfluxDBOrganization,
 		Category: flags.MetricsCategory,
 	}
+
+	OTELTracingFlag = &cli.BoolFlag{
+		Name:     "otel.tracing",
+		Usage:    "Enable OpenTelemetry tracing",
+		Category: flags.LoggingCategory,
+	}
+
+	OTELEndpointFlag = &cli.StringFlag{
+		Name:     "otel.endpoint",
+		Usage:    "OpenTelemetry collector endpoint (e.g., localhost:4317)",
+		Value:    "localhost:4317",
+		Category: flags.LoggingCategory,
+	}
+
+	OTELProtocolFlag = &cli.StringFlag{
+		Name:     "otel.protocol",
+		Usage:    "OpenTelemetry protocol: grpc or http",
+		Value:    "grpc",
+		Category: flags.LoggingCategory,
+	}
+
+	OTELInsecureFlag = &cli.BoolFlag{
+		Name:     "otel.insecure",
+		Usage:    "Use insecure connection to OpenTelemetry collector",
+		Value:    false,
+		Category: flags.LoggingCategory,
```

Comment:
Instead of these two flags, let's just support a URL as the endpoint and let the protocol of the URL specify the config.